cdk/src/open-data-platform/data-plane/schema/schema.sql

```diff
@@ -55,10 +58,22 @@ CREATE TABLE IF NOT EXISTS epa_violations(
     violation_code varchar(255) NOT NULL,
     compliance_status varchar(255) NOT NULL,
     start_date DATE NOT NULL,
+    end_date DATE NOT NULL,
     pws_id varchar(255) NOT NULL REFERENCES water_systems(pws_id),
     PRIMARY KEY(violation_id)
     );
 
+-- Violation counts per water system --
+
+CREATE VIEW violation_counts AS
```

Comment:
Does this need a `IF NOT EXISTS`? Or can it safely run multiple times even if the view already exists?